tests/integration_tests/test_integration_elasticsearchstore.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+import mock
+from mlflow_elasticsearchstore.elasticsearch_store import ElasticsearchStore
+
+
+@pytest.mark.integration_test
+def test_get_experiment():
+    store = ElasticsearchStore(store_uri="elasticsearch://elastic:password@localhost:9200",
+                               artifact_uri="viewfs://preprod-pa4/user/mlflow/mlflow_artifacts")
```

Comment:
You should create a fixture for that since you will use it for every integration test